src/app/content/utils.ts

```diff
@@ -20,17 +21,33 @@ export const getContentPageReferences = (content: string) =>
       };
     });
 
+const parseBookTree = (archiveBook: ArchiveBook) => {
+  archiveBook.tree.contents = archiveBook.tree.contents.map((subtree) => {
+    const domNode = new DOMParser().parseFromString(`<div id="container">${subtree.title}</div>`, 'text/html');
```

Comment:
Can we use somehow use `getArchiveTreeSectionTitle` or any other util to parse this title? Or maybe create new util for this?

Reply:
I've felt like the parsing of book tree should be ran once per book. I don't think the additional data in `.os-part-text` is necessary in anyway right now.

I'm not sure about `createTitle` in `seoUtils`. Should it have access to that data?

cc. @TomWoodward

Reply:
I guess we don't need this in any other place, but Tom is better to answer that

One more thing: please move new DOMParser() outside of `.map`